content/connectivity/akeneo/plugin.en.md

```diff
@@ -7,60 +7,232 @@ description: "Akeneo API integration in satellites pipelines"
+and which method you want to use to retrieve your data.
+
+The list of available tables is quite long and depends on the version of your Akeneo (enterprise or community).
+The options available are the same as for the [loader](#building-a-loader).
+
+The `merge` option allows you to add data to your dataset, in a sense merging your actual dataset with your new data.
+
+The `map` option comes from the [FastMap](../../../connectivity/fast-map) plugin, feel free to read its documentation
+to understand how to use it.
+
+```yaml
+akeneo:
+  lookup:
+    type: product
+    method: all
+    merge:
+      map:
+        - field: '[options]'
+          expression: 'lookup["code"]'
+  client:
+    api_url: 'http://demo.akeneo.com/'
+    client_id: '414yc7d9mnk044ko4wswgw80o8ssw80gssos488kk8ogss40ko'
+    secret: '4k8ee6n44m4gkkg0coc8o4w4coscw0w4cg0wg8sc0wsk0sw8gs'
+    username: 'demo_9573'
+    password: '516f3e3e5'
+```
+
+### Building a ConditionalLookup
+
+The conditional lookup is a lookup that takes conditions into account. Your lookup will be executed when each
+condition is met.
+
+About its configuration, you will find the same options as for the classic lookup, except that there is an additional
+`condition` option.
+
+```yaml
+akeneo:
+  lookup:
+    conditional:
+      - condition: '@=(input["type"] in ["pim_catalog_simpleselect", "pim_catalog_multipleselect"])'
+        type: attributeOption
+        code: '@=input["code"]'
+        method: listPerPage
+        search:
+          - { field: enabled, operator: '=', value: '@=input["code"]', scope: '@=input["code"]', locale: '@=input["fr_FR"]' }
+        merge:
+          map:
+            - field: '[options]'
+              expression: 'join("|", lookup["code"])'
+  # ...
+```
+
+### Building a loader
+
+In the configuration of your extractor, you must specify the type of table you are going to work on
+and which method you want to use to insert your data.
+
+The list of available tables is quite long et dépend de la version de votre Akeneo (enterprise ou community).
+
+For a community version, you can choose between several types of tables: `product`, `category`, `attribute`,
+`attributeOption`, `attributeGroup`, `family`, `productMediaFile`, `locale`, `channel`, `currency`, `measureFamily`,
+`associationType`, `familyVariant`, `productModel`.
+
+For an enterprise version, you will have in addition to the types present in the community version : `publishedProduct`,
+`productModelDraft`, `productDraft`, `asset`, `assetCategory`, `assetTag`, `referenceEntityRecord`, `referenceEntityAttribute`,
+`referenceEntityAttributeOption`, `referenceEntity`.
+
+For each table, the following 4 methods are available :
+
+- `create` : insert a row into the table
+- `upsert` : insert or update (if it already exists) a row into the table
+- `upsertList` : insert or update a list of data
+- `delete` : delete a row from the table
```

Comment:
```suggestion
- `create`: insert a new resource
- `upsert`: will try to update the resource, otherwise the resource will be created
- `upsertList`: will try to update a resources list, otherwise the resources will be created
- `delete`: delete a resource from the table
```